js/tests/keycodes-symphony-dom_test.mjs

```diff
@@ -1,20 +1,16 @@
 export const tests = []
 
-export const setup = async ({ page }) => ({
-  getNotes: async () =>
-    await page.$$eval('.note', (nodes) => {
-      return nodes.map((note) => note.textContent)
-    }),
-})
-
 const characters = `didyouhandlethekeydowneventcorrectly`
 
 tests.push(async ({ page, eq, ctx }) => {
   // check that a note is created and matches the right letter when a key is pressed
   for (const [i, character] of characters.split('').entries()) {
     await page.keyboard.down(character)
     const typed = characters.slice(0, i + 1).split('')
-    eq(await ctx.getNotes(), typed)
+    const getNotesTc = await page.$$eval('.note', (nodes) => {
```

Comment:
is there any special reason why the new function is `getNotesTc` instead of `getNotes`?

Reply:
Tc = TextContent (note.TextContent)